common/scripted_effects/06_scripted_effects_legendary_bloodlines.txt

```diff
@@ -909,8 +909,22 @@ spawn_old_duelist_effect = {
 				health = 2
 			}
 		}
-		
+
 		random_list = {
+			# Warcraft
+			10 = {
+				add_trait = marksman_captain_leader
+			}
+			10 = {
+				add_trait = aerial_raider_leader
+			}
+			10 = {
+				add_trait = artillery_engineer_leader
+			}
+			10 = {
+				add_trait = taskmaster_leader
```

Comment:
what if character is not evil slave driver?

Reply:
Taskmasters =/= slave driver. Updated localisation to better reflect this